addons/binding/org.openhab.binding.hyperion/src/main/java/org/openhab/binding/hyperion/internal/connection/JsonTcpConnection.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.hyperion.internal.connection;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.net.UnknownHostException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link JsonTcpConnection} class is responsible for handling
+ * the communication with the Hyperion json server.
+ *
+ * @author Daniel Walters - Initial contribution
+ */
+public class JsonTcpConnection {
+
+    private Logger logger = LoggerFactory.getLogger(JsonTcpConnection.class);
```

Comment:
should be `private static final`.

Reply:
Isn't the preference to have a non-static logger?